test/validator.test.ts

```diff
@@ -114,45 +114,44 @@ describe("validator", function () {
     });
 
     describe("without autoWait", function () {
-      let transactionHash: string = "";
-      this.beforeEach(async () => {
+      this.beforeAll(async () => {
         db.config.autoWait = false;
-        try {
-          const { meta } = await db
-            .prepare(`INSERT INTO ${txn.name}(name) VALUES(NULL);`)
-            .run();
-          transactionHash = meta.txn!.transactionHash!;
-          // This will throw
-          await meta.txn?.wait();
-        } catch (err) {
-          // no op
-        }
       });
+
+      this.afterEach(() => {
+        db.config.autoWait = true;
+      });
+
       test("when we call the receipt api and it returns an error", async function () {
+        const { meta } = await db
+          .prepare(`INSERT INTO ${txn.name} (name) VALUES(NULL);`)
+          .run();
+
+        const transactionHash = meta.txn!.transactionHash!;
+        // This will throw since not null has been violated, but we
+        // want to wait until the Validator has processed the txn
+        try {
+          await meta.txn?.wait();
+        } catch (e) {}
+
         const receipt = await api.receiptByTransactionHash({
           chainId,
           transactionHash,
         });
         match(receipt.error!, /.*msg: NOT NULL constraint failed:.*/);
       });
-      this.afterEach(() => {
-        db.config.autoWait = true;
-      });
-    });
 
-    describe("without autoWait", function () {
```

Comment:
This is not related to the PR, but I noticed that there seems to have been a copy-pasta gone wrong here.  There was an identical but separate `describe` for each of these tests, and two of the tests looked like they were testing the exact same thing.

Reply:
Huh, maybe a leftover merge conflict?